src/Components/Navbar/UserNavbar.js

```diff
@@ -76,23 +43,24 @@ export default function UserNavBar(props) {
       ].join('')
     }
   ];
-  const unauthedRoutes = [{ title: 'Events', route: '/events' },
-    { title: 'Discord Bot', route: '/discordSJSU' }];
+  const unauthedRoutes = [{ title: 'About', route: '/about' },
```

Comment:
part of suggestion 2

```suggestion
  const unauthedRoutes = [{ title: 'About', route: '/#about' },
```

then go into [`src/Components/Jumbotron/Jumbotron.js:21`](https://github.com/SCE-Development/Core-v4/blob/075e17067d90a8170da4e4b3ea3c54b4a92818b9/src/Components/Jumbotron/Jumbotron.js#L21) and do:

```html
<div id="about">
```

now, the link will take the user to the about section on the homepage.